docker-compose.build.yml

```diff
@@ -3,10 +3,11 @@ services:
   nginx:
     build: ./nginx
   api:
-    build: .
+    build:
+      target: api
```

Comment:
```suggestion
      context: .
      target: api
```

Unable to run `docker compose` without the `context` field

Reply:
huh, I had the issue the other way around (when context had subargs, target was needed)

Reply:
might be that here.. it does not have subargs :)